### ISSUE TITLE
paper icons need text labels

### ISSUE THREAD

**Issue by nickallan1** [scottyphillips/mitsubishi_echonet#9]
just installed the mitsubishi_hass custom component into hass.io and noticed that the icons don't have text labels on the dashboard. I'm totally blind and rely on a screen reader to speak what is happening, without text labels on the icons, my screen reader just says unlabelled button for everything.
Are you able to add text labels to the controls that are rendered in the lovelace UI? 
You can use the aria-label attribute to add text labels to the controls. a screen reader will then be able to associate a text label to the control. e.g aria-label="mode"

Info on aria-label and other info on making web ui's more accessible can be found at
https://www.w3.org/TR/WCAG21/

I appreciate anything you can do.

Nick

**Comment by scottyphillips** [comment]
Hi Nick, 

Thanks for installing the component and sorry to hear about your usability issue. Unfortunately any specific issues with the implementation of the Lovalace UI are more in the realm of the Home Assistant Devs themselves. My custom component adheres to their interface specifications for the Home Assistant Thermostat model and uses the standard defined modes (heating, cooling, fan) but the actual view is rendered by the Lovelace UI which isn't really touched by my component. 

What I will do is copy this issue over to the Home Assistant repo as it is highly likely that this issue impacts a lot of elements within the Lovelace UI and it should be sorted out. It shouldn't be too difficult for the Lovelace Devs to apply the aria-labels to all their UI elements and as good corporate citizens they absolutely must do this. 

Kind Regards,

Scott Phillips.

**Comment by scottyphillips** [comment]
Hi Nick,

I have mentioned this to the Home Assistant Polymer devs for issue 3267 (looks like you have raised some issues there too).   https://github.com/home-assistant/home-assistant-polymer/issues/3267

Kind regards, 

Scott Phillips

**Comment by nickallan1** [comment]
Hi @scottyphillips thanks for that. I should have realized that the icons in the lovelace UI weren't being rendered by your code. I appreciate you following it up with the hass project.
